codegen/template/Verifier.sol

```diff
@@ -6,7 +6,7 @@ import "./UncheckedMath.sol";
 contract Verifier is Plonk4VerifierWithAccessToDNext {
     using UncheckedMath for uint256;
 
-    function get_verification_key() public pure returns(VerificationKey memory vk) {
+    function get_verification_key() internal pure returns(VerificationKey memory vk){
```

Comment:
Please revert this change. It is important to have public function for getting verification keys, circuit breaker from server rely on it